lib/byron/test/integration/Cardano/Wallet/Byron/Config.hs

```diff
@@ -145,8 +145,7 @@ withConfig action =
               , (BlockchainParameters, NodeVersionData)
               )
     setupConfig = do
-        dir <- getCanonicalTemporaryDirectory
-            >>= \tmpRoot -> createTempDirectory tmpRoot "cardano-wallet-byron"
+        let dir = "/tmp/cardano-wallet-byron"
```

Comment:
The node was complaining about the socket path being too long. This is temporary.

Reply:
> The node was complaining about the socket path being too long. This is temporary.

What o.O ?

Reply:
Could you report (i.e. open a GH ticket) that to ouroboros-network ?

Reply:
👍 I'll verify I get the same error and report.

Reply:
```
{-# LINE 1062 "Network/Socket/Types.hsc" #-}
unixPathMax :: Int
unixPathMax = 108
{-# LINE 1064 "Network/Socket/Types.hsc" #-}

{-# LINE 1065 "Network/Socket/Types.hsc" #-}

-- We can't write an instance of 'Storable' for 'SockAddr' because
-- @sockaddr@ is a sum type of variable size but
-- 'Foreign.Storable.sizeOf' is required to be constant.

-- Note that on Darwin, the sockaddr structure must be zeroed before
-- use.

-- | Write the given 'SockAddr' to the given memory location.
pokeSockAddr :: Ptr a -> SockAddr -> IO ()

{-# LINE 1076 "Network/Socket/Types.hsc" #-}
pokeSockAddr p sa@(SockAddrUnix path) = do
    when (length path > unixPathMax) $ error "pokeSockAddr: path is too long"
    zeroMemory p $ fromIntegral $ sizeOfSockAddr sa
```

I suspect there's nothing they could do. Won't report.

https://unix.stackexchange.com/questions/367008/why-is-socket-path-length-limited-to-a-hundred-chars

Changing the `cardano-wallet-byron` subdir to `cw-byron` seems to do it though.